src/app/features/profile/store/profile.effects.ts

```diff
@@ -11,8 +11,8 @@ export class ProfileEffects {
     getProfile$ = createEffect(() =>
         this.action$.pipe(
             ofType(getProfilePending),
-            switchMap(() => this.usersService.getProfile()
-            .pipe(map((profile: IProfile) => getProfileSuccess({ profile }))))
+            switchMap(() => this.usersService.getUser('')
```

Comment:
passing here an empty string seems like a bad solution
just make it optional or set default value in a method
```
getUser (id: string = '')
```

```
getUser (id?: string) {
  const idPart = id ? `&${id}` : '';
}
```